docs/detections/shared-exception-lists.asciidoc

```diff
@@ -0,0 +1,131 @@
+[[shared-exception-lists]]
+== Create and manage shared exception lists
+
+Shared exception lists allow you to group exceptions together and then apply them to a set of rules. Use the Rule Exceptions page to set up shared exception lists.  
+
+[role="screenshot"]
+image::images/rule-exceptions-page.png[Rule exceptions page]
+
+[float]
+[[create-shared-exception-list]]
+=== Create shared exception lists
+
+Set up shared exception lists to contain exception items:
+
+. Go to *Manage* -> *Rule Exceptions*.
+. Click *Create shared exception list* -> *Create shared list*. 
+. Give the shared exception list a name. 
+. (Optional) Provide a description. 
+. Click *Create shared exception list*.
+
+[float]
+[[add-exception-items]]
+=== Add exception items to shared exception lists
+
+Create exception items for shared exception lists:
```

Comment:
Maybe consider cutting this line? It feels a but redundant with line 23, but also a little confusing because it's a bit different. I feel like line 23 could go right into the instructional bullet points starting on line 27.

Reply:
I also went back and forth on this for the reasons you listed. I ultimately decided to keep it so that it matched the structure the previous and following sections used (e.g., title, lead-in sentence, then steps). I'm still leaning towards keeping it to maintain that consistency, but agree with you that the lead-in sentence is unclear. How about something simpler like:

```suggestion
Add exception items:
```

Reply:
> I also went back and forth on this for the reasons you listed. I ultimately decided to keep it so that it matched the structure the previous and following sections used (e.g., title, lead-in sentence, then steps). I'm still leaning towards keeping it to maintain that consistency, but agree with you that the lead-in sentence is unclear. How about something simpler like:

@nastasha-solomon I like this option. 👍